_inc/client/traffic/index.jsx

```diff
@@ -64,6 +67,13 @@ export class Traffic extends React.Component {
 		return (
 			<div>
 				<QuerySite />
+				{
```

Comment:
What do you think of moving this downer on the page? It feels weird to put it above our paid features.

Reply:
Makes sense, done 👍